packages/xo-web/src/common/render-xo-item.js

```diff
@@ -408,7 +412,7 @@ const xoItemToRender = {
   // Pool objects.
   'VM-template': ({ id }) => <VmTemplate id={id} />,
   'VM-template-resourceSet': ({ id }) => <VmTemplate id={id} self />,
-  host: ({ id }) => <Host id={id} />,
+  host: ({ id, memoryFree }) => <Host id={id} memoryFree={memoryFree} />,
```

Comment:
I don't think this is a good idea. It's too much effort to use it and it requires to build a new object every time (`{ ...host, memoryFree: true }`). Open to discussion, though.

Reply:
I added some changes and I avoided `({ ...host, memoryFree: true })`